debian/patches/series

```diff
@@ -31,5 +31,6 @@ ubuntu_privacy_hide_buggy_controls.patch
 pop_allow_sound_above_100.patch
 pop_shop.patch
 pop_appearance.patch
-pop_hidpi.patch
```

Comment:
I believe we want to keep the patches in alphabetical order. Or at least that is normally quite the pet peeve of @jackpot51 's.

Reply:
@brs17 no, the order of patches must be adjusted so they apply cleanly, not so they are alphabetical. This reordering is fine